server/src/main/java/com/cloud/storage/VolumeApiServiceImpl.java

```diff
@@ -224,6 +225,8 @@ public class VolumeApiServiceImpl extends ManagerBase implements VolumeApiServic
     @Inject
     private VolumeDetailsDao _volsDetailsDao;
     @Inject
+    private VolumeGroupDao _volsGroupDao;
```

Comment:
I don't like or see the use of this `_` naming convention, let's break with it! (no :-1: on this PR in any way!)

Reply:
Yep, I don't like it either, it's a little bit JavaScript style and this also not my favorite, but since I'm new I try to follow the existing source code. 

I will change this :)